src/io/flutter/view/FlutterView.java

```diff
@@ -373,6 +401,37 @@ private void debugActiveHelper(FlutterApp app, @Nullable InspectorService inspec
 
     toolWindow.setIcon(ExecutionUtil.getLiveIndicator(FlutterIcons.Flutter_13));
 
+    if (JxBrowserManager.getInstance().isInstalled()) {
+      final DevToolsManager devToolsManager = DevToolsManager.getInstance(app.getProject());
+
+      if (devToolsManager.hasInstalledDevTools()) {
+        addBrowserInspectorViewContent(app, inspectorService, toolWindow);
+      }
+      else {
+        final CompletableFuture<Boolean> result = devToolsManager.installDevTools();
+        result.thenAccept(o -> addBrowserInspectorViewContent(app, inspectorService, toolWindow));
+      }
+
+      onAppChanged(app);
+
+      app.addStateListener(new FlutterApp.FlutterAppListener() {
```

Comment:
why is this code needed here? this seems copied from somewhere and not clearly relevant.